src/devices/dell_xps_13_9345.c

```diff
@@ -24,7 +24,7 @@ static EFI_GUID dell_inc__xps_13_9345_hwids[] = {
 
 static struct device dell_inc__xps_13_9345_dev = {
 	.name  = L"Dell Inc. XPS 13 9345",
-	.dtb   = L"qcom\\x1e80100-dell-tributo-13.dtb", /* Tentative. */
+	.dtb   = L"qcom\\x1e80100-dell-xps13-9345.dtb", /* Tentative. */
```

Comment:
Oh! Seems like I've missed this when checking for merged dts a couple of days ago... Thanks for noticing and updating this!

Since the name is now canonical, let's drop the note here and also remove italic in the readme for it

```suggestion
	.dtb   = L"qcom\\x1e80100-dell-xps13-9345.dtb",
```

Reply:
Ah yes. missed it.